### DISCUSSION TITLE
Can sequence support MySQL full-text indexing? Does Op.match currently only support PG

### DISCUSSION THREAD

**Discussion by lwmxiaobei** [sequelize/sequelize#16266] [Help & Questions] [answered]
Can sequence support MySQL full-text indexing? Does Op.match currently only support PG ？
<img width="490" alt="image" src="https://github.com/sequelize/sequelize/assets/16284775/30a5b19a-7eb5-4fd2-acc0-488c98085a6b">

**Comment by WikiRik** [comment, marked as answer]
You can use a literal for this. See the following example;
```js
  await Foo.findAll({
    where: and(
      literal('MATCH (`time`) AGAINST (\'abc\' IN NATURAL LANGUAGE MODE)'),
      { time: '14' },
    ),
  });
```
